.github/workflows/master.yml

```diff
@@ -240,7 +240,7 @@ jobs:
       - name: Install dependencies
         run: |
           source ./dev/install-common-deps.sh
-          pip install pyspark torch transformers
+          pip install pyspark torch transformers langchain
```

Comment:
Do we want to make `langchain` a common deps? It seems to me that langchain should not be required for non-genai users. (related to my comment in `mlflow/models/evaluation/base.py`)

Can we test if langchain is not installed, a user should still be able to call `mlflow.evaluate()`?

Reply:
It should not be a common dependency - this line change is only for test infra. 
That being said, a +1 to the validation check that the patch function with evaluate will work if langchain is not installed. Temporarily removing the install from the REPL context would be good to validate.